oic.r.cred.raml

```diff
@@ -95,8 +118,19 @@ traits:
         description: The request is invalid.
 
   post:
+
```

Comment:
Yes, all of the schema changes required for CR-50 have to be made.  I was under the impression that you were doing that... in your infinite spare time!!  ;)

Seriously, if you can't do it, we need to raise the flag on the next Sec WG call so that someone else can be assigned to it.